src/quilla/steps/validations/xpath.py

```diff
@@ -267,6 +271,22 @@ def _check_visual_parity(self) -> ValidationReport:
             )
 
         if len(plugin_result) == 0:
```

Comment:
When will plugin_result return empty list VS None? Why creating new baseline image needs empty list return of plugin_result?

Reply:
It will return None if the plugin is not configured, it will return an empty byte object (`b''`) when there is a plugin configured but there is no matching baseline